apps/heft/UPGRADING.md

```diff
@@ -141,8 +190,9 @@ The following is an example "heft.json" file defining both a "build" and a "test
           }
         },
         "copy-assets": {
-          "taskEvent": {
-            "eventKind": "copyFiles",
+          "taskPlugin": {
+            "packageName": "@rushstack/heft",
+            "pluginName": "copy-files-plugin",
```

Comment:
I think it makes sense to allow this for now since it's still brand new and this is intended to be an overview of the new version of Heft. However, I do agree in general that this shouldn't be changed retroactively. Ideally this should be the last breaking schema change.